src/chttpd/test/chttpd_db_test.erl

```diff
@@ -187,6 +190,32 @@ should_return_409_for_put_att_nonexistent_rev(Url) ->
     end).
 
 
+should_return_update_seq_when_set_on_all_docs(Url) ->
+    ?_test(begin
+        [create_doc(Url, "testdoc" ++ ?i2l(I)) || I <- lists:seq(1, 3)],
+        {ok, RC, _, RespBody} = test_request:get(Url ++ "/_all_docs/"
```

Comment:
Getting `/home/vagrant/git/release/couchdb/src/chttpd/test/chttpd_db_test.erl:196: Warning: variable 'RC' is unused` here.
Maybe add `?assertEqual(200, RC)` here and below to catch possible bad reply from  `/_all_docs`?